src/modules/interface/powertoy_system_menu.h

```diff
@@ -0,0 +1,19 @@
+#pragma once
+
+#include <string>
+
+class PowertoyModuleIface;
+
+class PowertoySystemMenuIface
+{
+public:
+  /* 
+   * Set configuration of system menu items for specific powertoy module. Configuration
+   * parameters such as item name (and hotkey), item status at creation (enabled/disabled)
+   * and whether check box will appear next to item name when action is taken, are passed
+   * as JSON formatted string.
+   */
+  virtual void SetConfiguration(PowertoyModuleIface* module, const wchar_t* config) = 0;
```

Comment:
json is needed in the module settings API since the data is sent to a webview control and packing it in a json is the easier way to send it.
But for this API we only process the data in the runner and I don't think we need a json format, that will save exta processing and make the code cleaner.